github3/github.py

```diff
@@ -24,6 +24,7 @@
 
 from .decorators import (requires_auth, requires_basic_auth,
                          requires_app_credentials)
+from .utils import timestamp_parameter
```

Comment:
With the exception of our decorators, everything else ipmorts a module instead of a function or class. Please follow the same pattern here.

Reply:
Sure thing! I believe I was led astray by this other example in the code base: https://github.com/sigmavirus24/github3.py/blob/bb40e7e89441f9d2c084e4a26fb4a2a9a0a4c305/github3/issues/__init__.py#L10